### PR TITLE
copy images from ghcr and public.ecr.aws for IPv6 support

### PR DESCRIPTION
**Pull request by DockToFuture**
**What this PR does / why we need it**:
Copy images from ghcr and public.ecr.aws for IPv6 support of gardener shoot clusters.
**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: config/images/images.yaml [hunk 1]
```diff
@@ -142,4 +142,29 @@ images:
   tags:
   - 2.8.1
   - 2.8.2
-  - 2.8.3
\ No newline at end of file
+  - 2.8.3
+# ebs-csi-driver/aws-ebs-csi-driver
+- source: public.ecr.aws/ebs-csi-driver/aws-ebs-csi-driver
```
Comment (ialidzhikov) [inline config/images/images.yaml:147]: In https://github.com/gardener/ci-infra/pull/748#pullrequestreview-1469215924 we discussed that we only copy DockerHub images. Can you elaborate now why we need to copy GHCR and ECR images?

cc @oliver-goetz

**Review by ialidzhikov — CHANGES_REQUESTED** [review]

File: config/images/images.yaml (hunk 1 above)
Comment (ialidzhikov) [reply config/images/images.yaml:147]: Additionally, the aws-ebs-csi-driver has a image in registry.k8s.io - ref https://github.com/kubernetes-sigs/aws-ebs-csi-driver#container-images. I don't really understand why we need to copy it to our GCR.

Comment (DockToFuture) [reply config/images/images.yaml:147]: In an IPv6 only shoot cluster you can not resolve IPv4 addresses thus it is not possible to download the images. It's required to support IPv6 in gardener.

Comment (ialidzhikov) [reply config/images/images.yaml:147]: As shared in https://github.com/gardener/ci-infra/pull/982/files#r1379797871, please drop aws-ebs-csi-driver and use the `registry.k8s.io/provider-aws/aws-ebs-csi-driver` image in provider-aws.

**Review by ialidzhikov — CHANGES_REQUESTED** [review]

File: config/images/images.yaml [hunk 2]
```diff
@@ -142,4 +142,24 @@ images:
...
+# credativ/plutono
+- source: ghcr.io/credativ/plutono
+  destination: eu.gcr.io/gardener-project/3rd/credativ/plutono
+  tags:
+  - v7.5.25
+# prometheus-operator/prometheus-config-reloader
+- source: ghcr.io/prometheus-operator/prometheus-config-reloader
+  destination: eu.gcr.io/gardener-project/3rd/prometheus-operator/prometheus-config-reloader
+  tags:
+  - v0.67.1
```
Comment (ialidzhikov) [inline config/images/images.yaml:165]: prometheus-config-reloader will be tackled by https://github.com/gardener/gardener/pull/8751. You could revert.

**Review by ScheererJ — COMMENTED** [review]
/lgtm

File: config/images/images.yaml [hunk 3]
```diff
@@ -142,4 +142,19 @@ images:
...
+# credativ/valitail
+- source: ghcr.io/credativ/valitail
+  destination: eu.gcr.io/gardener-project/3rd/credativ/valitail
+  tags:
+  - v2.2.8
+# credativ/plutono
+- source: ghcr.io/credativ/plutono
+  destination: eu.gcr.io/gardener-project/3rd/credativ/plutono
+  tags:
+  - v7.5.25
```
Comment (ialidzhikov) [inline config/images/images.yaml:160]: Move this to L72 as this is where images from `gardener/imagevector/images.yaml` are referenced.

**Review by ialidzhikov — CHANGES_REQUESTED** [review]

**Comment by ialidzhikov** [comment]
/lgtm

in case you don't want to wait for the credativ colleagues

Shouldn't we also add copy for `public.ecr.aws/eks/aws-load-balancer-controller`?

**Comment by oliver-goetz** [comment]
Why is this PR open for such a long time? 🤔  The versions are not even the most current one anymore.

**Comment by DockToFuture** [comment]
I've updated the versions, can we get this in now or do you have any objections?

**Review by ScheererJ — COMMENTED** [review]
/lgtm

**Review by ialidzhikov — CHANGES_REQUESTED** [review]
> Shouldn't we also add copy for `public.ecr.aws/eks/aws-load-balancer-controller` and `public.ecr.aws/ebs-csi-driver/volume-modifier-for-k8s`?

ping for the above question

**Comment by DockToFuture** [comment]
I've added them.

File: config/images/images.yaml [hunk 4]
```diff
@@ -66,6 +66,21 @@ images:
...
+# credativ/valitail
+- source: ghcr.io/credativ/valitail
+  destination: eu.gcr.io/gardener-project/3rd/credativ/valitail
+  tags:
+  - v2.2.13
+# credativ/plutono
+- source: ghcr.io/credativ/plutono
+  destination: eu.gcr.io/gardener-project/3rd/credativ/plutono
+  tags:
+  - v7.5.28
```
Comment (ialidzhikov) [inline config/images/images.yaml:83]: These are all images related to gardener/gardener's images.yaml file (https://github.com/gardener/ci-infra/blob/092dc4b86602d0e05988eb74839995f5d5c94caa/config/images/images.yaml#L2-L5). Hence no need of the comments.

```suggestion
- source: ghcr.io/credativ/vali
  destination: eu.gcr.io/gardener-project/3rd/credativ/vali
  tags:
  - v2.2.13
- source: ghcr.io/credativ/valitail
  destination: eu.gcr.io/gardener-project/3rd/credativ/valitail
  tags:
  - v2.2.13
- source: ghcr.io/credativ/plutono
  destination: eu.gcr.io/gardener-project/3rd/credativ/plutono
  tags:
  - v7.5.28
```

**Review by ialidzhikov — COMMENTED** [review]
Final nit, otherwise lgtm

File: config/images/images.yaml [hunk 5]
```diff
@@ -66,6 +66,18 @@ images:
   tags:
   - v1.24.1
   - v1.26.4
+- source: ghcr.io/credativ/vali
+  destination: eu.gcr.io/gardener-project/3rd/credativ/vali
```
Comment (ialidzhikov) [inline config/images/images.yaml:70]: Aa, I was about to miss the most important. GCR domain has to be switched with the AR.

PS: Sorry for not noticing earlier.

**Review by ialidzhikov — APPROVED** [review]
/lgtm

File: config/images/images.yaml (hunk 5 above)
Comment (DockToFuture) [reply config/images/images.yaml]: I've updated the path.

**Review by ialidzhikov — COMMENTED** [review]
/lgtm

**Review by oliver-goetz — APPROVED** [review]
awesome 🥳

**Pull request merged**